js/js-inputs-and-strings/js-inputs-and-strings_switch-values/js/index.js

```diff
@@ -24,3 +26,18 @@ uppercaseButton.addEventListener("click", () => {
 lowercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toLowerCase();
 });
+
+uppercaseButton.addEventListener("click", () => {
+  secondInput.value = secondInput.value.toUpperCase();
+});
+
+lowercaseButton.addEventListener("click", () => {
+  secondInput.value = secondInput.value.toLowerCase();
+});
+
+switchButton.addEventListener("click", () => {
+  let extraCup = firstInput.value;
```

Comment:
Not sure what `extraCup` means 🤔 
```suggestion
  let firstInputValue = firstInput.value;
```